src/PrestaShopBundle/Form/Admin/Sell/Product/Image/ImageDropzoneType.php

```diff
@@ -43,6 +47,48 @@
  */
 class ImageDropzoneType extends TranslatorAwareType
 {
+    /**
+     * @var FeatureInterface
+     */
+    private $multistoreFeature;
+
+    public function __construct(
+        TranslatorInterface $translator,
+        array $locales,
+        FeatureInterface $multistoreFeature
+    ) {
+        parent::__construct($translator, $locales);
+        $this->multistoreFeature = $multistoreFeature;
+    }
+
+    public function buildForm(FormBuilderInterface $builder, array $options)
```

Comment:
don't we need "@inhertiDoc" here or the return type? (I remember in cases when code inherits the return type from  older interface which has no types, we used to add that, or I think it won't break anything if you just add a return type here